mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -158,6 +175,413 @@ properties:
+              finding is included in the response.
+          - !ruby/object:Api::Type::Enum
+            name: 'minLikelihood'
+            description: |
+              Only returns findings equal or above this threshold. See https://cloud.google.com/dlp/docs/likelihood for more info
+            values:
+              - :VERY_UNLIKELY
+              - :UNLIKELY
+              - :POSSIBLE
+              - :LIKELY
+              - :VERY_LIKELY
+            default_value: :POSSIBLE
+          - !ruby/object:Api::Type::NestedObject
+            name: 'limits'
+            description: Configuration to control the number of findings returned.
+            properties:
+              - !ruby/object:Api::Type::Integer
+                name: 'maxFindingsPerItem'
+                description: Max number of findings that will be returned for each item
+                  scanned. The maximum returned is 2000.
+                required: true
+              - !ruby/object:Api::Type::Integer
+                name: 'maxFindingsPerRequest'
+                description: Max number of findings that will be returned per request/job.
+                  The maximum returned is 2000.
+                required: true
+              - !ruby/object:Api::Type::Array
+                name: 'maxFindingsPerInfoType'
+                description: |
+                  Configuration of findings limit given for specified infoTypes.
+                item_type: !ruby/object:Api::Type::NestedObject
+                  properties:
+                    - !ruby/object:Api::Type::NestedObject
+                      name: 'infoType'
+                      required: true
+                      description: |
+                        Type of information the findings limit applies to. Only one limit per infoType should be provided. If InfoTypeLimit does
+                        not have an infoType, the DLP API applies the limit against all infoTypes that are found but not
+                        specified in another InfoTypeLimit.
+                      properties:
+                        - !ruby/object:Api::Type::String
+                          name: 'name'
+                          required: true
+                          description: |
+                            Name of the information type. Either a name of your choosing when creating a CustomInfoType, or one of the names listed
+                            at https://cloud.google.com/dlp/docs/infotypes-reference when specifying a built-in type.
+                        - !ruby/object:Api::Type::String
+                          name: 'version'
+                          description: |
+                            Version of the information type to use. By default, the version is set to stable
+                    - !ruby/object:Api::Type::Integer
+                      name: 'maxFindings'
+                      description: Max findings limit for the given infoType.
+                      required: true
```

Comment:
Just want to double check if this field is required.

Reply:
I checked this field manually along with `infoType` and both aren't required. But as they are part of array, I couldn't add `AtleastOneOf`. And keeping the block empty works, won't be passed in POST call. Seems fine to remove required from both.